### ISSUE TITLE
Cannot seem to get "application_image' to work 

### ISSUE THREAD

**Issue by mlanser** [gpressutto5/laravel-slack#5]
First, I love this package! As far as I can tell it seems to work great. But I cannot seem to get the 'application_image' to work. I'm not sure what I'm doing wrong.

The image is stored in:
   `<docroot>/public/images/logos/main-logo.png`

and I'm setting the config value as follows:
   `'application_image' => '/images/logos/main-logo.png'`

I must be missing something, but don't know what.

TIA,
-martin.

**Comment by gpressutto5** [comment]
Hey Martin, sorry for the delay. As the description says, it should be a valid URL. So you could use something like this: _(not tested)_
```php
'application_image' => asset('images/logos/main-logo.png')
```

**Comment by mlanser** [comment]
Sorry, but I still cannot get this to work. I feel I've tried every possible combination including hardcoding the path. 

By the way, `asset('images/logos/main-logo.png')` generates the following error: 

`Argument 2 passed to Illuminate\Routing\UrlGenerator::__construct() must be an instance of Illuminate\Http\Request, null given`

I verified that I have a copy of the image in `resources/assets/images/logos/main-logo.png`

I think I have tried all helper functions at this point and I have also tried all path combinations using both 'resources' and 'public' sub-folders. 

Finally I tried to emulate your test cases `testSendMessageWithTheDefaultSlackImage()` and `testSendMessageWithTheDefaultConfigImage()`. But even with the hardcoded paths (which I first checked in a browser), it still doesn't seem to work. 

So ... at this point I'm not sure what else to try. I must be missing something very simple, but I cannot figure out what it is.

Any thoughts or suggestions?

TIA,
-martin.
